src/main/java/core/basesyntax/service/strategy/BalanceHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Fruit;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void getOperation(FruitDao fruitDao, String fruit, int quantity) {
```

Comment:
1) Please do not combine `void` and `get` in name. It's impossible to read such code.
If you use getOperation name that you expect that this method returns you some operation but it provides you nothing.
performOperation could be more appropriate but again you don't pass any operation, you pass some service, fruit and quatity, the is no operation anywhere around.
Due to that fact I don't have any idea what is going on here unless I deep-dive into several methods.

2) Do not pass services you rely on as method arguments. They should be class fields and initialized through constructor